helmctlr/helm.go

```diff
@@ -101,6 +104,7 @@ func (c Controller) ResourceUpdated(oldR, newR *unstructured.Unstructured) {
 		return
 	}
 	metrics.UpdatedReleases.Inc()
+	metrics.LastSuccessfulUpdate.Set(float64(time.Now().UTC().UnixNano()) / 1000000000)
```

Comment:
Can we not use `.Unix()` instead of `.UnixNano() / 1000000000`? `Unix()` returns seconds

Reply:
I was originally using .Unix(), but needed more precision for the tests.